build/Linux+BSD/portable/copy-libs

```diff
@@ -72,7 +74,7 @@ getCrossPlatformDependencies() {
   copyLib libQt5Sensors.so.5
   copyLib libQt5Sql.so.5
   copyLib libQt5Svg.so.5
-  copyLib libQt5WebChannel.so.5
```

Comment:
lasconic tells me on IRC that "we dont' use any function from qtwebchannel currently".  

So it seems that this isn't an issue for the time being. I suppose even though check-depends reports this dependency, we don't need to worry about that missing dependency, since musescore still runs fine without it...until someone decides to implement something with it.